x/gamm/simulation/operations.go

```diff
@@ -84,7 +84,7 @@ func genFuturePoolGovernor(r *rand.Rand, addr sdk.Address, tokenList []string) s
 }
 
 func genPoolAssets(r *rand.Rand, acct simtypes.Account, coins sdk.Coins) []types.PoolAsset {
-	numCoins := 2 + r.Intn(Min(coins.Len(), 6))
+	numCoins := Max(2, r.Intn(Min(coins.Len(), 6)))
```

Comment:
Hrm, we want this to be `numCoins ~ U[2, 8]`

Reply:
We may need to change our account generation function to give more coin types to each person